src/transformers/models/mask2former/image_processing_mask2former.py

```diff
@@ -424,6 +424,13 @@ def __init__(
             self._max_size = kwargs.pop("max_size")
         else:
             self._max_size = 1333
+        if "reduce_labels" in kwargs:
+            warnings.warn(
+                "The `reduce_labels` parameter is deprecated and will be removed in a future version. Please use"
+                " `do_reduce_labels` instead.",
```

Comment:
We should specify a version here 
```suggestion
                "The `reduce_labels` parameter is deprecated and will be removed in v4.43. Please use"
                " `do_reduce_labels` instead.",
```